scripts/prepublish.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Ensure that we're publishing from the master branch.
+ */
+
+const simpleGit = require('simple-git');
+const chalk = require('chalk');
+
+const git = simpleGit();
+
+(async () => {
+  await git.init();
+  const branchName = (await git.branch()).current;
```

Comment:
Unless `simple-git` is expected to be used in more places, I think this could be accomplished with a:

```js
execSync('git rev-parse --abbrev-ref HEAD').toString().trim()
```

Don't feel too strongly either way, though.

Reply:
Ah, fair point. I was _this_ close to just writing another bash script, but refrained myself. This seems like a good compromise. :+1: